vaccine_feed_ingest/runners/in/arcgis/normalize.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import os
+import pathlib
+import re
+import sys
+from typing import List, Optional
+
+# import schema
+site_dir = pathlib.Path(__file__).parent
+state_dir = site_dir.parent
+runner_dir = state_dir.parent
+root_dir = runner_dir.parent
+sys.path.append(str(root_dir))
+
+from schema import schema  # noqa: E402
+
+# Configure logger
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(levelname)s:%(name)s:%(message)s",
+    datefmt="%m/%d/%Y %H:%M:%S",
+)
+logger = logging.getLogger("ak/arcgis/normalize.py")
+
+
+def _get_id(site: dict) -> str:
+    data_id = site["attributes"]["GlobalId"]
+
+    # Could parse these from directory traversal, but do not for now to avoid
+    # accidental mutation.
+    site = "arcgis"
+    runner = "in"
+
+    # Could parse these from the input file name, but do not for now to avoid
+    # accidental mutation.
+    arcgis = "97135bbb1bec488e9717aca061c03e41"
+    layer = 0
+
+    return f"{runner}:{site}:{arcgis}_{layer}:{data_id}"
+
+
+def _get_inventory(site: dict) -> Optional[List[schema.Vaccine]]:
+    vaccines_field = site["attributes"]["Vaccine_Type"].lower().split(",")
+
+    potentials = {
+        "Pfizer": schema.Vaccine(vaccine="pfizer"),
+        "Moderna": schema.Vaccine(vaccine="moderna"),
+        "Johnson & Johnson": schema.Vaccine(vaccine="janssen"),
+        "FRPP": schema.Vaccine(vaccine="frpp"),
```

Comment:
"FRPP stands for ["Federal Retail Pharmacy Partnership"](https://www.cdc.gov/vaccines/covid-19/retail-pharmacy-program/index.html), and is not a vaccine type.

If a row has this data, let's avoid setting the type of the vaccine.